packages/dds/merge-tree/src/opBuilder.ts

```diff
@@ -97,6 +99,30 @@ export function createObliterateRangeOp(start: number, end: number): IMergeTreeO
 	};
 }
 
+/**
+ * Creates the op to obliterate a range
+ *
+ * @param start - The start of the range to obliterate
+ * @param end - The end of the range to obliterate
+ *
+ * @internal
+ */
+export function createObliterateRangeOpSided(
+	start: SequencePlace,
+	end: SequencePlace,
+): IMergeTreeObliterateSidedMsg {
+	const startPlace = normalizePlace(start);
+	const endPlace =
+		typeof end === "number"
```

Comment:
I'd rather not introduce any endpoint string behavior, as it is not tested anywhere. for instance, shouldn't this be string?